src/app/app.config.ts

```diff
@@ -64,6 +64,7 @@ export interface UiConfig {
 
 export interface EditionConfig {
     title: string;
+    manifestURL: string; // TODO: check name and position for this property
```

Comment:
I would move `manifestUrl` configuration from `EditionConfig` interface to `FileConfig`. 
Also, why not use camel case for property name?

Reply:
Ok for the `FileConfig`. 

Usually it is considered camel case since URL is an acronym